### PR TITLE
rename experimental and dump to chrome and collect

### PR DESCRIPTION
**Pull request by soulgalore**

### CODE REVIEW

**Comment by tobli** [comment]
Should we perhaps parse the old names for a while, print a warning a convert to the new?

File: lib/support/cli.js [hunk 1]
```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
...
-      describe: 'Dump Chromes traceCategories log to disk.',
+      describe: 'Collect Chromes traceCategories log to disk.',
       group: 'chrome'
     })
     .option('chrome.traceCategories', {
       describe: 'Set the trace categories.',
       type: 'string',
       group: 'chrome'
     })
+    .option('chrome.collectChromePerflog', {
```
Comment (tobli) [inline lib/support/cli.js]: Now that it's in the Chrome category, I suggest we shorten it to chrome.collectPerflog.

File: lib/support/cli.js [hunk 2]
```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
...
+      type: 'boolean',
+      describe: 'Collect Chromes performance log with Network.* events and save to disk.',
+      group: 'chrome'
+    })
+    .option('chrome.collectChromeNetlog', {
```
Comment (tobli) [inline lib/support/cli.js]: Same as above, chrome.collectNetlog

File: lib/support/cli.js [hunk 3]
```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
       describe: 'Choose which device to use. If you do not set it, random device will be used.',
       group: 'chrome'
     })
-    .option('chrome.dumpTraceCategoriesLog', {
+    .option('chrome.collectTraceCategoriesLog', {
```
Comment (tobli) [inline lib/support/cli.js]: Either use a lower case L for log, or use upper case for Perflog and Netlog.

File: lib/support/cli.js [hunk 4]
```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
...
       type: 'boolean',
-      describe: 'Dump Chromes traceCategories log to disk.',
+      describe: 'Collect Chromes traceCategories log to disk.',
```
Comment (tobli) [inline lib/support/cli.js]: My understanding is that this means "include trace events in the perfLog". I suggest renaming the option and description to something that reflects that. Also, should this imply chrome.collectPerfLog? Maybe also chrome.traceCategories should imply this option (and by extension chrome.collectPerfLog).

**Comment by soulgalore** [comment]
Taking PRs for the PR!

**Comment by tobli** [comment]
If it's ok with you I can push to the PR branch as well.

**Comment by soulgalore** [comment]
yes please that will make it much faster, it's so rare that we disagree :)

**Pull request merged**